TriblerGUI/widgets/trustpage/js/main.js

```diff
@@ -2,6 +2,10 @@
  * This file creates an SVG element and draws the network using it.
  */
 
+// TODO: think about how to split this in different javascipt files
+// TODO: start using JSDoc and setup some CI tool to check
+// TODO: choose a testing framework and start writing tests
```

Comment:
I don't think these `TODO`'s belong in the codebase. I suggest you create tickets for these.